src/practitest_firecracker/core.clj

```diff
@@ -29,17 +29,16 @@
     (if exit-message
       (exit (if ok? 0 1) exit-message)
       (do
-        (remove-bom (file (first (:reports-path options))))
+        (doall (map remove-bom (:reports-path options)))
```

Comment:
If you need to do something for each element of a collection, you can use `doseq`.
```clojure
(doseq [path (:report-paths options)]
  (remove-bom path))
```